test/HostActivationTests/project.json

```diff
@@ -1,16 +1,15 @@
 {
   "version": "1.0.0-*",
-
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0-rc3-004338"
+      "version": "1.0.0"
     },
```

Comment:
This seems to be pulling from stage0 when it runs which is currently pulling the 1.0.0 stable version which is why I changed it to that. As far as I can tell this isn't running on the shared framework we are producing.